### PR TITLE
Get rid of temporary files

### PR DESCRIPTION
**Pull request by saper**
Thanks to a true MSBuild Mastery(R)

### CODE REVIEW

File: win/libsass.vcxproj [hunk 1]
```diff
@@ -239,7 +238,7 @@
     <ClCompile Include="..\src\values.cpp" />
     <ClCompile Include="..\sassc\sassc.c" />
   </ItemGroup>
-  <ItemGroup Condition="$(VisualStudioVersion) &lt; 14.0">
+  <ItemGroup Condition="$([MSBuild]::Subtract('$(VisualStudioVersion)', 14.0)) &lt; 0">
```
Comment (am11) [inline win/libsass.vcxproj]: Is it not building with VS2013 without this change? When I tested, it was building with that simpler condition in both VS2015 and VS2013.

File: win/libsass.vcxproj [hunk 2]
```diff
@@ -5,15 +5,14 @@
...
-    </ReadLinesFromFile>
-    <Exec Command="del *-version" />
+    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags"
+      ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="LIBSASS_VERSION" />
+    </Exec>
+    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags"
+      ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="SASSC_VERSION" />
+    </Exec>
```
Comment (am11) [inline win/libsass.vcxproj]: That is pretty neat! :)
nit: Please put `ConsoleToMSBuild="true">` slugs in same line as `<Exec..`, as it does not exceed the git preference of 120 characters (its 100 characters with `ConsoleToMSBuild..` in same line).

File: .gitignore [hunk 3]
```diff
@@ -1,77 +1,33 @@
...
-
-# Cloned testing dirs
-sassc/
-sass-spec/
-
-installer/
+/missing
+/stamp-h1
+/compile
+/sassc
```
Comment (am11) [inline .gitignore]: This looks like a significant change. IMO, we should drop this change for the on-going PR and submit a separate PR for it.

**Comment by am11** [comment]
@saper, thanks! I have left few comments. Other than that, it is an excellent idea to avoid IO engagement as much as possible! :+1:

File: .gitignore (hunk 3 above)
Comment (saper) [inline .gitignore]: On Fri, 14 Aug 2015, Adeel Mujahid wrote:

> > -
> > -# Final results
> > -
> > -sassc++
> > -libsass.la
> > -src/support/libsass.pc
> > -
> > -# Cloned testing dirs
> > -sassc/
> > -sass-spec/
> > -
> > -installer/
> > +/missing
> > +/stamp-h1
> > +/compile
> > +/sassc
> 
> This looks like a significant change. IMO, we should drop this change for the on-going PR and submit a separate PR for it.

This was not intended.

File: win/libsass.vcxproj (hunk 2 above)
Comment (saper) [inline win/libsass.vcxproj]: On Fri, 14 Aug 2015, Adeel Mujahid wrote:

> > -    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags &gt; sassc-version" />
> > -    <ReadLinesFromFile File="sassc-version">
> > -      <Output TaskParameter="Lines" PropertyName="SASSC_VERSION" />
> > -    </ReadLinesFromFile>
> > -    <ReadLinesFromFile File="libsass-version">
> > -      <Output TaskParameter="Lines" PropertyName="LIBSASS_VERSION" />
> > -    </ReadLinesFromFile>
> > -    <Exec Command="del *-version" />
> > +    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags"
> > +      ConsoleToMSBuild="true">
> > +      <Output TaskParameter="ConsoleOutput" PropertyName="LIBSASS_VERSION" />
> > +    </Exec>
> > +    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags"
> > +      ConsoleToMSBuild="true">
> > +      <Output TaskParameter="ConsoleOutput" PropertyName="SASSC_VERSION" />
> > +    </Exec>
> 
> nit: Please put `ConsoleToMSBuild="true">` slugs in same line as `<Exec..`, as it does not exceed the git preference of 120 characters (its 100 characters with `ConsoleToMSBuild..` in same line).

Come on, no more that, say 80 lines...

File: win/libsass.vcxproj (hunk 1 above)
Comment (saper) [inline win/libsass.vcxproj]: On Fri, 14 Aug 2015, Adeel Mujahid wrote:

> > @@ -239,7 +238,7 @@
> >      <ClCompile Include="..\src\values.cpp" />
> >      <ClCompile Include="..\sassc\sassc.c" />
> >    </ItemGroup>
> > -  <ItemGroup Condition="$(VisualStudioVersion) &lt; 14.0">
> > +  <ItemGroup Condition="$([MSBuild]::Subtract('$(VisualStudioVersion)', 14.0)) &lt; 0">
> 
> Is it not building with VS2013 without this change? When I tested, it was building with that simpler condition in both VS2015 and VS2013.

Let me check again... when I tested it before it didn't

File: win/libsass.vcxproj (hunk 2 above)
Comment (am11) [inline win/libsass.vcxproj]: I can give you two reasons (ahem):

* It matches the rest of the file formatting. 
* If in future, someone upgrades this project for vNext of VS, it will not format this document again as part of the process, and hence produce undesired diffs: https://github.com/sass/libsass/pull/1439#discussion_r36974694.

**Comment by saper** [comment]
In my testing yesterday I did something wrong and couldn't get `&lt; 14.0` to work, but now it's fine.

So I re-submitted only the `<Exec>` change.

**Comment by am11** [comment]
Perfect! :tada:
